modules/acquia_cms_site_studio/composer.json

```diff
@@ -30,7 +30,7 @@
             "file-mapping": {
                 "[web-root]/drush/Commands/ConfigImportExportCommands.php": "modules/acquia_cms_site_studio_cm/src/Commands/ConfigImportExportCommands.php",
```

Comment:
@chandan-singh7929 Since this is also scaffolding file, can we move it under same directory i.e `assets/scaffold/files` ?

Reply:
Done